### PR TITLE
Vagrantfile: use criu from stable repo

### PR DESCRIPTION
**Pull request by kolyshkin**
CRIU 3.14 has made its way to the F32 stable repo, let's use it.

### CODE REVIEW

**Comment by AkihiroSuda** [comment]
LGTM

File: Vagrantfile [hunk 1]
```diff
@@ -13,17 +13,14 @@ Vagrant.configure("2") do |config|
     v.cpus = 2
   end
   config.vm.provision "shell", inline: <<-SHELL
-    curl -OSs https://kojipkgs.fedoraproject.org/packages/criu/3.14/1.fc32/x86_64/criu-3.14-1.fc32.x86_64.rpm
     cat << EOF | dnf -y shell
 config exclude kernel,kernel-core
 config install_weak_deps false
-localinstall criu-3.14-1.fc32.x86_64.rpm
 update
-install iptables gcc make golang-go libseccomp-devel bats jq git-core
```
Comment (AkihiroSuda) [inline Vagrantfile:22]: Can we pin the version?

Comment (kolyshkin) [reply Vagrantfile:22]: > Can we pin the version?

Yes (`criu = 3.14` or something like that) but I don't see the reason why. If the next criu release fails, we want to know about it (and can certainly use a previous release once we do)

**Comment by AkihiroSuda** [comment]
@mrunalp @cyphar PTAL

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**